### ISSUE TITLE
Proposal for new option: autostart Pomodoro

### ISSUE THREAD

**Issue by pnauta** [schmich/marinara#322]
It would be really great of Pomodoro could be set to start immediately when the browser is started.  In my case it is Chrome.  I mostly forget to start it, and nomally only start it one hour later.

**Comment by armandohimself** [comment]
I agree with pnauta. I've set my browser to open the pomodoro page on a new tab upon startup as a work around.

**Comment by silviuk** [comment]
+1

**Comment by YousufAzadSami** [comment]
+1

**Comment by mofahmin** [comment]
+1

**Comment by Ryan-Efendy** [comment]
> I agree with pnauta. I've set my browser to open the pomodoro page on a new tab upon startup as a work around.

@armandohimself how do you set it as your new tab?

**Comment by aliarousyoucef** [comment]
@Ryan-Efendy  entre settings and copy past the URL to home page for example: 

chrome-extension://.../modules/options.html#/settings

**Comment by armandohimself** [comment]
@Ryan-Efendy Click on 3 dots in top right corner of chrome browser.

Click settings

Scroll down till you see on startup

![image](https://user-images.githubusercontent.com/20542956/105812790-cd4ea980-5f7c-11eb-8d18-11ee0f022d15.png)
